server/driverTripProposalsRoute.js

```diff
@@ -0,0 +1,65 @@
+import getDrivers from '../spreadsheetDatabase/getDrivers.js'
+
+export const PASSAGER_CONTACT_DIRECT_ACCEPT = true;
+export const PASSAGER_CONTACT_DIRECT_REFUSE = false;
+const PASSAGER_CONTACT_DIRECT_NO_ANSWER = PASSAGER_CONTACT_DIRECT_REFUSE; // qui ne dit mot... ne consent pas
```

Comment:
@DavidBruant (je fais ce commentaire après avoir mergé la PR car la fonctionnalité passe avant)

Ces variables m'ont un peu perturbé : étant donné qu'elles ne sont pas partagées par plusieurs blocs de code, je pense que j'aurais compris plus facilement si `true`ou `false` (ainsi que le principe d'opt-in) étaient renvoyés directement dans le test ligne 19, la propriété `Contact direct pasager` étant très claire. Encore qu'elle pourraît même être appelée `Contact direct passager autorisé`.
Je ne vois pas non plus l'intérêt du `PASSAGER_CONTACT_DIRECT_ACCEPT` utilisé dans direct/server.js, mais peut-être que j'ai loupé quelque chose.